Assets/!/Prefabs/Guns/Trails/Bullet Trail.prefab

```diff
@@ -157,17 +157,17 @@ TrailRenderer:
       m_RotationOrder: 4
     colorGradient:
       serializedVersion: 2
-      key0: {r: 0.1981132, g: 0.1981132, b: 0.1981132, a: 1}
-      key1: {r: 1, g: 1, b: 1, a: 1}
-      key2: {r: 0, g: 0, b: 0, a: 0}
+      key0: {r: 0.764151, g: 0.04884557, b: 0, a: 1}
```

Comment:
Is there a reason you changed the bullet trail? The bullet trail data is configured in the scriptable object so this won't matter.

Reply:
No idea how that happend, I swapped out the knife prefab for the ak47 perfab for better testing and somehow maybe this happend. This can be discarded as it was completly accidental.